modules/authorization/proc-determining-permission-policy-and-role-configuration-source.adoc

```diff
@@ -0,0 +1,35 @@
+[id='proc-determining-policy-and-role-source']
+= Determining permission policy and role configuration source
+
+You can configure {product} policy and roles by using different sources.
+To maintain data consistency, {product-short} associates each permission policy and role with one unique source.
+You can only use this source to change the resource.
+
+The available sources are:
+
+Configuration file::
```

Comment:
Is something missing here?

Reply:
Nothing missing, but I removed the two empty lines that might have been confusing.